sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -10,58 +10,29 @@
 
 namespace Azure.AI.TextAnalytics
 {
-    internal static class Transforms
+    internal static partial class Transforms
     {
-        #region Common
-
-        public static readonly Regex _targetRegex = new Regex("#/tasks/(keyPhraseExtractionTasks|entityRecognitionPiiTasks|entityRecognitionTasks|entityLinkingTasks|sentimentAnalysisTasks|extractiveSummarizationTasks|customSingleClassificationTasks|customMultiClassificationTasks|customEntityRecognitionTasks)/(\\d+)", RegexOptions.Compiled, TimeSpan.FromSeconds(2));
+        #region Needs Review
```

Comment:
We should consider auditing anything with the `Internal` suffix to see if it is still needed with the language API - they're not used by the legacy integration.